lib/components/tabs.js

```diff
@@ -29,35 +29,35 @@ export default class Tabs extends Component {
       }
       {
         tabs.length > 1 ?
-          [
-            <ul
-              key="list"
-              className={css('list')}
-              >
-              {
-                tabs.map((tab, i) => {
-                  const {uid, title, isActive, hasActivity} = tab;
-                  const props = getTabProps(tab, this.props, {
-                    text: title === '' ? 'Shell' : title,
-                    isFirst: i === 0,
-                    isLast: tabs.length - 1 === i,
-                    borderColor,
-                    isActive,
-                    hasActivity,
-                    onSelect: onChange.bind(null, uid),
-                    onClose: onClose.bind(null, uid)
-                  });
-                  return <Tab key={`tab-${uid}`} {...props}/>;
-                })
-              }
-            </ul>,
-            isMac && <div
-              key="shim"
-              style={{borderColor}}
-              className={css('borderShim')}
-              />
-          ] :
-          null
+        [
+          <ul
+            key="list"
+            className={css('list')}
+            >
+            {
+              tabs.map((tab, i) => {
+                const {uid, title, isActive, hasActivity} = tab;
+                const props = getTabProps(tab, this.props, {
+                  text: title === '' ? 'Shell' : title,
+                  isFirst: i === 0,
+                  isLast: tabs.length - 1 === i,
+                  borderColor,
+                  isActive,
+                  hasActivity,
+                  onSelect: onChange.bind(null, uid),
+                  onClose: onClose.bind(null, uid)
+                });
+                return <Tab key={`tab-${uid}`} {...props}/>;
+              })
+            }
+          </ul>,
+          isMac && <div
+            key="shim"
+            style={{borderColor}}
+            className={css('borderShim')}
+            />
+        ] :
+        null
```

Comment:
Looks like it's deps issue. `yarn` is kinda broken or outdated on master as `yarn install` changes current `yarn.lock` and adds new one in `app` folder.